poetry.lock

```diff
@@ -101,6 +101,24 @@ category = "dev"
 optional = false
 python-versions = "!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*,!=3.5.*,!=3.6.*,>=2.7"
 
+[[package]]
```

Comment:
I've always been curious: do we want the `poetry.lock` file to be this big and updated? The `python-versions` seems wrong to start with.

Reply:
It helps to recreate PyProject if something is different between each instance.